doc/devel/development_setup.rst

```diff
@@ -127,6 +127,8 @@ The simplest way to do this is to use either Python's virtual environment
         <file folder location>\Scripts\activate.bat  # Windows cmd.exe
         <file folder location>\Scripts\Activate.ps1  # Windows PowerShell
 
+      You may need to type python3 instead of python.
```

Comment:
Let's keep this targeted, so that other user can directly skip this (-> less mental load)

```suggestion
      On Mac, you may need to type python3 instead of python.
```